uint8 k_ep
uint8 k_run
float32 reward
float32 reward_avg
uint8 n_rollouts